### ISSUE TITLE
auto-test of word sets completeness

### ISSUE THREAD

**Issue by ruv** [gerryjackson/forth2012-test-suite#23]
Could this test suite also include a functionality to automatically check what the word sets are provided and what names from them are absent?
So, in place of  "word is not found" error, the testing programme will be able to skip the corresponding test, remeber this fact for the report, and proceed with the next test.

**Comment by gerryjackson** [comment]
Yes that is an idea we've had before and of course it can be added provided somebody puts in some effort to do it. There are other things that probably take priority such as covering all word sets.
